### PR TITLE
Adds dev dependencies

### PR DESCRIPTION
**Pull request by mschoettle**
* Adds dev dependencies (`flake8`, `flake8-isort`, `flake8-quotes`, `mypy`, `pip-tools`) for consistent coding style
* adds [editorconfig](https://editorconfig.org/)
* unifies coding style across all `.py` files

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.{py,rst,ini}]
+indent_style = space
+indent_size = 4
```
Comment (merwok) [inline .editorconfig:13]: reST blocks should use 3-space indents (and code inside reST code blocks 4-space, of course)

File: README.md [hunk 2]
```diff
@@ -12,10 +12,11 @@ I'd be happy to help if you'd like to make a version of this for your own city.
...
 1. Clone this repository then navigate to the checkout
-2. Create a virtual environment: `python3.8 -m venv .venv`
-3. Activate the environment: `. .venv/bin/activate` (the command is `.`, meaning source or include)
-4. Install dependencies: `pip install -r requirements.txt`
-5. Run the app!  `python -m app`
+1. Create a virtual environment: `python3.8 -m venv .venv`
+1. Activate the environment: `. .venv/bin/activate` (the command is `.`, meaning source or include)
+1. Install dependencies: `pip install -r requirements.txt`
+1. For development: Install development dependencies `pip install -r requirements-dev.txt`
+1. Run the app!  `python -m app`
```
Comment (merwok) [inline README.md]: Is this change required?  It makes the source text less readable in my opinion, which is too bad for a plain-text format.

File: requirements.in [hunk 3]
```diff
@@ -3,6 +3,6 @@ pandas == 1.0.*
 gunicorn
 requests
 lxml
-bs4
+beautifulsoup4
```
Comment (merwok) [inline requirements.in:6]: Ah good catch!  (it was working because the author added a `bs4` package on PyPI that depends on `beautifulsoup4`)

**Review by merwok — COMMENTED** [review]
Just some notes; I don’t know if the project lead is interested in enforcing a cosmetic style guide.

File: .editorconfig (hunk 1 above)
Comment (mschoettle) [reply .editorconfig:13]: Oups, I copied this from another project. Since they are not used here I could just remove the `rst` extension here.

File: README.md (hunk 2 above)
Comment (mschoettle) [reply README.md]: Thanks for your feedback. I am planning to make some more contributions so I thought it might make sense to add that. But I'm fine if there's no interest.

> Is this change required? It makes the source text less readable in my opinion, which is too bad for a plain-text format.

Ideally, if there's dependencies for different environments, a single command would be nice. Unfortunately, `pip-tools` does not seem to add `-r requirements.txt` at the top of `requirements-dev.txt`.

Comment (merwok) [reply README.md]: I was only asking about the change in the numbered list here!

Comment (mschoettle) [reply README.md]: Oh, sorry! Good point, was only thinking about how it will be rendered. I reverted that and made the dependency installation a one-liner.

File: README.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ Here are the steps to follow if you want to run COVID-19 MTL locally for testing
 1. Clone this repository then navigate to the checkout
 2. Create a virtual environment: `python3.8 -m venv .venv`
 3. Activate the environment: `. .venv/bin/activate` (the command is `.`, meaning source or include)
-4. Install dependencies: `pip install -r requirements.txt`
+4. Install dependencies: `pip install -r requirements.txt requirements-dev.txt`
```
Comment (merwok) [inline README.md]:
```suggestion
4. Install dependencies: `pip install -r requirements.txt -r requirements-dev.txt`
```

Comment (mschoettle) [reply README.md]: Good catch!

**Comment by jeremymoreau** [comment]
This is good to go I think, thanks!! I will merge it now. I'm fine with the proposed flake8 style linting. I also like having NumPy style docstrings if possible, but don't think we need to be super strict about enforcing code style beyond the automated linting.
